still_alive_publish_duration: 60
